.github/workflows/reusable-build.yml

```diff
@@ -15,6 +15,9 @@ name: Reusable build
 
 on:
   workflow_call:
+    secrets:
+      CODECOV_TOKEN:
+        required: true

```

Comment:
Why do we need this?
I saw this PR: https://github.com/notaryproject/notation-core-go/pull/199
Looks like for some reason it didn't work?

Reply:
Please have a look: https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow